vercel.json

```diff
@@ -12,7 +12,9 @@
   ],
   "redirects": [
     { "source": "/", "destination": "/docs" },
-    { "source": "/docs/apis/storage", "destination": "/docs/apis/preferences" }
+    { "source": "/docs/apis/storage", "destination": "/docs/apis/preferences" },
+    { "source": "/blog", "destination": "https://ionic.io/blog/tag/capacitor" },
```

Comment:
This line is redundant. It will redirect correctly if only the next line is there.

Reply:
yeah, my plan was to change the ones with slug to show the .md files in capacitor-site repository since they are not published anywhere, so that's why there are two entries, but I never got to change that